INTITA/Models/SpecialisationsList.swift

```diff
@@ -0,0 +1,63 @@
+//
+//  SpecialisationsList.swift
+//  INTITA
+//
+//  Created by Viacheslav Markov on 11.02.2021.
+//
+
+import Foundation
+
+enum SpecialisationsList: Int {
+    case programming = 1
+    case web_design
+    case manual_qa
+    case dev_ops
+    case system_administration
+    case hosting_support
+    case english_for_it
+    case hackers_school
+    case it_camp
+    case hr_and_ta
+    case project_management
+    case automation_qa
```

Comment:
Suggest you setting a rawValue for each case, not to be confused if the order will change.

Reply:
or add tests..
Change names to camel case

Reply:
These cases were deleted. 
The test is added.